Sources/Helpers/LayerSize.swift

```diff
@@ -8,6 +8,29 @@
 
 import Foundation
 
+/// Represents an axis within a LayerSize
+public enum LayerSizeAxis {
+    case f
+    case w
+    case h
+
+    static var all: [LayerSizeAxis] { return [.f, .w, .h] }
+
+    static func fromTF(index: Int) -> LayerSizeAxis? {
```

Comment:
Yes, but I wanted to explicitly show that those integer indexes make sense only when converting from TensorFlow